UACloudLibraryServer/Views/Shared/_Layout.cshtml

```diff
@@ -33,7 +33,7 @@
 
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2022 - UACloudLibrary - <a href="https://github.com/OPCFoundation/UA-CloudLibrary/tree/@ThisAssembly.GitCommitId" target="_blank">@ThisAssembly.AssemblyInformationalVersion</a> - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> 
+            &copy; 2022 - Opc.Ua.Cloud.Library - <a href="https://github.com/OPCFoundation/UA-CloudLibrary/tree/@ThisAssembly.GitCommitId" target="_blank">@ThisAssembly.AssemblyInformationalVersion</a> - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> 
```

Comment:
This one I would change to OPC Foundation.

Reply:
I changed it to OPC Foundation